src/utils/subject.ts

```diff
@@ -15,15 +18,22 @@ import {
   getIssue,
   getIssueOrPullRequestComment,
   getPullRequest,
+  getPullRequestReviews,
   getRelease,
 } from './api/client';
-import { fetchDiscussion, getLatestDiscussionComment } from './helpers';
+import {
+  fetchDiscussion,
+  getLatestDiscussionComment,
+  getTokenForHost,
+} from './helpers';
 
 export async function getGitifySubjectDetails(
   notification: Notification,
-  token: string,
+  accounts: AuthState,
```

Comment:
Why did we move the `getTokenForHost` call to here? Not that it matters much, just trying to understand the reasoning

Reply:
Reverted @afonsojramos - thanks for picking this up